.gitattributes

```diff
@@ -0,0 +1 @@
+cardano-cli/test/cardano-cli-test/files/input/example_anchor_data.txt -text
```

Comment:
?

Reply:
This is necessary because `git` automatically changes the carriage returns of text files. So it was changing `\n` into `\r\n` for Windows, and that changed the hash of the file, which in turn made the Windows CI tests fail